core/src/main/java/com/linecorp/armeria/server/AbstractServiceBindingBuilder.java

```diff
@@ -93,7 +93,7 @@
      * @throws IllegalArgumentException if the specified path pattern is invalid
      */
     public AbstractServiceBindingBuilder path(String pathPattern) {
-        defaultPathMapping = PathMapping.of(requireNonNull(pathPattern, "pathPattern"));
+        defaultRouteBuilder = Route.builder().path(pathPattern);
```

Comment:
nit: `requireNonNull(pathPattern, "pathPattern")`?

Reply:
Forgot to put it back. 😅